### PR TITLE
EPWW Split

### PR DESCRIPTION
**Pull request by ThaweK**
Fixing #160, also #166 included.

### CODE REVIEW

**Comment by PL-VACC-ACCPL3** [comment]
Changes authorized by PLVACC.

_Confirmation email sent to Nestor_

**Review by Connorseds2002 — APPROVED** [review]
As approved by vacc

File: FIRBoundaries.dat [hunk 1]
```diff
@@ -4444,223 +4444,571 @@ ENOB|0|0|20|63.05266|0.000000|82.000000|30.000000|72.500000|15.000000
...
+50.4104|18.2711
+50.4255|18.2941
+50.534|18.4418
+50.5643|18.5118
+51.0349|19.084
+51.21|19.4506
+51.2945|20.1347
+52.16575|20.967122
+52.1736|21.5551
+52.1117|23.2416
```
Comment (NelisV) [inline FIRBoundaries.dat:4924]:
```suggestion
```
duplicate vertex

File: FIRBoundaries.dat [hunk 2]
```diff
@@ -4444,223 +4444,571 @@ ENOB|0|0|20|63.05266|0.000000|82.000000|30.000000|72.500000|15.000000
...
+52.283333|23.166667
+52.266667|23.183333
+52.233333|23.183333
+52.183333|23.416667
+52.183333|23.45
+52.183333|23.483333
+52.15|23.5
+52.083333|23.65
+51.983333|23.683333
+52.1117|23.2416
```
Comment (NelisV) [inline FIRBoundaries.dat:4729]:
```suggestion
```
duplicate vertex

File: FIRBoundaries.dat [hunk 3]
```diff
@@ -4444,223 +4444,571 @@ ENOB|0|0|20|63.05266|0.000000|82.000000|30.000000|72.500000|15.000000
...
+50.666667|15.866667
+50.75|15.816667
+50.733333|15.75
+50.816667|15.45
+50.766667|15.4
+51.0|15.25
+51.016667|15.0
+51.3|15.033333
+51.416667|14.966667
+51.483333|14.933333
```
Comment (NelisV) [inline FIRBoundaries.dat:4994]:
```suggestion
```
duplicate vertex

**Review by NelisV — CHANGES_REQUESTED** [review]
Jup looks good! neatly aligned :). 
if you can remove these 3 duplicate points, i'll approve.

File: FIRBoundaries.dat (hunk 3 above)
Comment (DoubleWhisky) [reply FIRBoundaries.dat:4994]: Seems like another error in original data... We will fix it.

**Comment by DoubleWhisky** [comment]
@NelisV I compared entry vs final data and those points appeared somwhere in QGIS or format transformation with your script. I'll try to redo every step and find out what's wrong.

**Review by NelisV — APPROVED** [review]
Thanks! looks good now 👍

**Review by Adrian2k — APPROVED** [review]

**Pull request merged**